runner/runner.go

```diff
@@ -118,23 +99,25 @@ func (r *Runner) Run(ctx context.Context, query ...string) error {
 		)
 		switch engine {
 		case "shodan":
-			agent, err = shodan.NewWithOptions(&uncover.AgentOptions{RateLimiter: shodanRateLimiter})
+			agent, err = shodan.New()
 		case "censys":
-			agent, err = censys.NewWithOptions(&uncover.AgentOptions{RateLimiter: censysRateLimiter})
+			agent, err = censys.New()
 		case "fofa":
```

Comment:
@xm1k3 , it depends on your preference , but I think we can remove `.New()` functions since they are not used anymore and only decrease overall code coverage. I think we can directly append to array
```go
agents = append(agents, &publicwww.Agent{})
```